app/javascript/components/mentoring/mentor_conversations_list.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import dayjs from 'dayjs'
+import RelativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(RelativeTime)
+
+function MentorConversationListRow({
```

Comment:
There was probably a decision to use JS instead of TS, but here you're going to benefit from type annotation this at least with jsdoc, in some editors, such as vscode and rubymine.

Reply:
@iHiD will know why JS was chosen over TS.